tests/end_to_end/features/steps/autovariants.py

```
@@ -26,31 +26,56 @@
-def step_impl(context, transformation_name):
-    @context.spark.df_transformation(
-        name=transformation_name,
-        inputs=[context.transactions],
-    )
-    def some_transformation(df):
-        """Unedited transactions"""
-        return df
+@when(
+    'I register "{transformation_name}" transformation with auto variant of type "{tf_type}"'
+)
+def step_impl(context, transformation_name, tf_type):
```

Ruff (0.8.2)

Redefinition of unused `step_impl` from line 23

(F811)